app/controllers/api/v1/code_schools_controller.rb

```diff
@@ -42,10 +42,19 @@ def destroy
         end
       end
 
+      def get_moocs
+        mooc_school = CodeSchool.where(is_mooc: "true")
+        if mooc_school
```

Comment:
This conditional will return false positives.  If the ActiveRecord collection comes back empty, it will still satisfy the check, and would render an empty set of data.  For example:

```ruby
[13] pry(main)> schools = CodeSchool.where(id: 99999)
=> []
[11] pry(main)> if schools
[11] pry(main)*   p 'something'
[11] pry(main)* else
[11] pry(main)*   p 'nada'
[11] pry(main)* end
"something"
=> "something"
```

To account for this, one option would be to use the `.present?` method:

```ruby
[14] pry(main)> schools.present?
=> false
```

That all said, since this is a straight `GET` without any params, I think in this case we can safely just render the results, and remove the conditional completely.  That way, if there are results, they will be rendered.  And if there are no results, then that will be reflecting in the rendering, as well.

And there's really no error handling you would need to account for here.